module-4/backend/week_one.md

```diff
@@ -1,21 +1,73 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+1. What's the most useful thing you learned from completing the intermission week work?
 
-Note: When you're done, submit a PR. 
+I liked working through the Express REPL exercises.
 
-1. What's the most useful thing you learned from completing the intermission week work?
 2. What are some tools to help debug JavaScript code?
+
+- Debugger
+- Console (in browser)
+- PryJS
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+- Mocha
+- Chai
+- NPM / Webpack
+
 4. What is the syntax for invoking a function?
-5. What's the difference between `==` and `===` in JavaScript?
-6. What's the difference between asynchronous and synchronous JavaScript? 
-7. What's a callback function and what are some reasons when we use/need callback functions?
-8. What's the biggest difference between a promise and a callback?
-9. How do we setup a route when creating an API with Node and Express?
-10. What's `npm` and what do we use it for?
+
+`()`, so if you have a function like:
+```javascript
+function() {...}
```

Comment:
Nitpicky, but there would still need to be `()` on the end of the function to invoke it. `function () {}()`.